.gitignore

```diff
@@ -36,12 +36,12 @@ venv
 build
 deepinv.egg-info
 dist
-datasets
+examples/datasets
```

Comment:
Is this removing the examples from git tracking?

Reply:
initially it removed every "datasets" folder in the git tracking
notably the "deepinv.datasets" folder
and I changed it to examples/datasets, which contains raw dataset

Reply:
i though that the initial behavior was intended for "examples/datasets" -> do i need to change this line ?